src/Bridges/Nette/TracyExtension.php

```diff
@@ -108,17 +108,20 @@ public function afterCompile(Nette\PhpGenerator\ClassType $class)
 
 		if ($this->debugMode) {
 			foreach ($this->config->bar as $item) {
-				if (is_string($item) && substr($item, 0, 1) === '@') {
-					$item = new Nette\DI\Statement(['@' . $builder::THIS_CONTAINER, 'getService'], [substr($item, 1)]);
-				} elseif (is_string($item)) {
-					$item = new Nette\DI\Statement($item);
+				if (is_string($item)) {
+					if ($item[0] ?? '' === '@') {
+						$item = new Nette\DI\Statement(['@' . $builder::THIS_CONTAINER, 'getService'], [substr($item, 1)]);
+					} else {
+						$item = new Nette\DI\Statement($item);
+					}
 				}
 				$initialize->addBody($builder->formatPhp(
 					'$this->getService(?)->addPanel(?);',
 					Nette\DI\Helpers::filterArguments([$this->prefix('bar'), $item])
 				));
 			}
 
+			assert(\class_exists(Nette\Http\Session::class));
```

Comment:
it's better to add nette/http as dev dependency

Reply:
Added.